django_project_base/templates/bootstrap/project-list.html

```diff
@@ -6,7 +6,7 @@
                 <li class="nav-item dropdown">
                     <i class="nav-link fas fa-th-list fa-2x" data-toggle="dropdown"></i>
                     <div class="dropdown-menu" aria-labelledby="navbarDropdown">
-                        <a v-for="(project, idx) in projectList" v-on:click="projectSelected(project.id)"
+                        <a v-for="(project, idx) in projectList" v-on:click="projectSelected(project.slug)"
```

Comment:
tule nadvse upam, da je projectList lazy-loading var?

Reply:
ko je komponenta videa se podatki naložijo in zrendrajo